packages/core/src/codewhisperer/commands/startSecurityScan.ts

```diff
@@ -34,15 +34,27 @@ import { openUrl } from '../../shared/utilities/vsCodeUtils'
 import { AuthUtil } from '../util/authUtil'
 import { DependencyGraphConstants } from '../util/dependencyGraph/dependencyGraph'
 import path from 'path'
+import { once } from '../../shared/utilities/functionUtils'
 
 const performance = globalThis.performance ?? require('perf_hooks').performance
-const securityScanOutputChannel = vscode.window.createOutputChannel('CodeWhisperer Security Scan Logs')
-const codeScanLogger = makeLogger({
-    outputChannels: [securityScanOutputChannel],
-})
```

Comment:
Using memoization (`once()`) eliminates these top-level variables, and improves their type signatures (eliminates nullability).